hphp/test/zend/good/ext/standard/tests/strings/strncmp_variation6.php.expectf

```diff
@@ -771,6 +771,6 @@ int(0)
 int(0)
 
 -- Checking with out of character's range, given in binary format --
-int(1)
+bool(true)
```

Comment:
Ok.  I think the RE will have to depend on the preceding comment.  Lots of 'int(0)' in
the file.